applications/CoSimulationApplication/custom_utilities/conversion_utilities.cpp

```diff
@@ -0,0 +1,48 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//                   Ashish Darekar
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/variables.h"
+#include "utilities/parallel_utilities.h"
+#include "utilities/variable_utils.h"
+#include "utilities/atomic_utilities.h"
+#include "conversion_utilities.h"
+
+namespace Kratos {
+
+//void ConversionUtilities::ConvertElementalDataToNodalData(ModelPart& rModelPart)
+void ConversionUtilities::ConvertElementalDataToNodalData(ModelPart& rModelPart, Variable<array_1d<double,3> >& rVariable)
```

Comment:
cool!
But if we make the input of the variable necessary, then we also need them for nodal and elemental var:

```suggestion
void ConversionUtilities::ConvertElementalDataToNodalData(
    ModelPart& rModelPart, 
    const Variable<array_1d<double,3> >& rElementalVariable,
    const Variable<array_1d<double,3> >& rNodalVariable)
```

Reply:
Ya, True, Previously I thought its just a name `FORCE`, but it is a **Variable** `FORCE`. I have made changes. 
So, While calling this, I need to use like this
`ConversionUtilities::ConvertElementalDataToNodalData(test_model_part, FORCE, FORCE);`
right ?

Reply:
yes exactly

Reply:
done